GetPhrase.js

```diff
@@ -0,0 +1,26 @@
+var CreatePhrase = require("./CreatePhrase");
+var fs = require("fs");
+
+var GetPhrase = function(Word) {
+  this.arr = [];
+  fs.readFile("phrases.txt", "utf8", function(err, phrases) {
+    if (!err) {
+      this.output = phrases.split(",");
+      for (var i = 0; i < output.length; i++) {
+        this.arr.push(output[i]);
+      }
+    } else {
+      console.log("Error: " + err);
+    }
+    this.randPhrase = this.arr[Math.floor(Math.random() * this.arr.length)];
```

Comment:
I wouldn't make this random as it can be annoying if as a user I get the same phrase to play multiple times in a row which is possible.